### ISSUE TITLE
Github Issue templates

### ISSUE THREAD

**Issue by RocketNinja15** [chunky-dev/chunky#735]
Add templates for issues like I have here. if you go into my .github/ISSUE_TEMPLATE you can see both of the .md files. It auto labes them as a bug or enhancement, and separate templates for each one when you create a issue. You can select, suggestion or bug whenever you make an issue. Try it out

https://github.com/RocketNinja15/Cheatin-Network

**Comment by leMaik** [comment]
This was proposed before but tbh I don't see much value yet. 🤔 Happy to discuss this though.

Related #699

**Comment by RocketNinja15** [comment]
Yes, well, it could help organize your issues without much work at all, this at maximum will take 10 minutes. It might save you in the long run.

**Comment by jackjt8** [comment]
I will have a look into this shortly. It was part of the code of conduct and contributing guides on other repos so I should try to create a template
